openquake/qa_tests_data/classical/case_5/job.ini

```diff
@@ -17,7 +17,7 @@ number_of_logic_tree_samples = 0
 
 # km
 rupture_mesh_spacing = 0.01
-complex_fault_mesh_spacing = 0.01
+complex_fault_mesh_spacing = 0.1
```

Comment:
No, I kept them different on purpose. This calculation is dominated by the  `complex_fault_mesh_spacing` since it has a complex fault.